akka-actor/src/main/scala/akka/serialization/Serializer.scala

```diff
@@ -78,6 +78,7 @@ trait Serializer {
 
 object Serializers {
 
+  // NOTE!!! Before making changes to this method please check DaemonMsgCreateSerializer.serialize method whether it will work correctly after them.
```

Comment:
? it's not used in DaemonMsgCreateSerializer?

Reply:
yes, but the logic is almost the same. If you would change the logic here, it is likely that it will need to be changed in `DaemonMsgCreateSerializer.serialize`

Reply:
I mean the determining the manifest. It can probably be described better. 
`If you change this method it is likely that DaemonMsgCreateSerializer.serialize needs the changes too`

Reply:
ah, I see what you mean now